nym-vpn-apple/Home/Sources/Home/HomeViewModel.swift

```diff
@@ -350,6 +351,16 @@ private extension HomeViewModel {
         return isInstalledAndRunning
     }
 
+    func needsHelperUpdate() async -> Bool {
+        do {
+            let version = try await grpcManager.version()
+            guard helperManager.requiredVersion != version else { return false }
```

Comment:
This guard seems unnecessary, could just `return helperManager.requiredVersion != version`